lib/chef/knife/azure_server_create.rb

```diff
@@ -69,8 +69,7 @@ def load_winrm_deps
 
       option :ssh_port,
         :long => "--ssh-port PORT",
-        :description => "The ssh port. Default is 22.",
-        :default => '22'
+        :description => "The ssh port. Default is 22. If --azure_connect_to_existing_dns set then default SSH port is random"
```

Comment:
@adamedx , why should underscores in the proper name of the setting be removed? It is far easier to search for such items by copy/paste the string, eg I found it is spelled right as per https://github.com/opscode/knife-azure/blob/master/README.md ?